### ISSUE TITLE
[2.3.2RC1] Different collapse behaviour of single fields when designing a section

### ISSUE THREAD

**Issue by jurajkapsz** [symphonycms/symphony-2#1687]
**Situation:**
When creating a new section, gradually added fields are not collapsible on the go.

Only after saving the section (or with a saved section), the collapse feature is available. This is a change from previous SCMS versions.

**Proposal:**
Single fields could be collapsible from the start as before, even with the new "Collapse all" feature not available from the beginning (it would require a save). When I am creating a complex section, its nice to have the fields collapsed on the go.

Ok, it makes sense to save the work in progress, but this non clickable behaviour is kind of not expected. But maybe its only a matter of a habit.
